cpp/src/binaryop/compiled/binary_ops.cu

```diff
@@ -99,10 +99,10 @@ struct apply_binop_scalar_rhs_lhs : apply_binop<Rhs, Lhs, Out> {
 };
 
 template <typename Lhs, typename Rhs, typename Out>
-struct binary_op {
+struct binary_ope {
```

Comment:
Would it be better to `use binary_operator` ?

Reply:
That is what I thought as well, however I tried `binary_operation` and I got a name conflict with something else. I figure `_ope` was next best option as it is what we use in `kernel.cpp`.

Reply:
I really dislike this. "ope" is neither a word nor a common abbreviation. And it's easy to pronounce differently than the thing you are shortening. 3 strikes.  Plus it's unnecessary if you improve `unary_operator` rather than impairing `binary_operator`.

Reply:
Fixed.